runtimes/ruby/ruby-on-rails.md

```diff
@@ -52,10 +52,12 @@ it from your environment, like you would with `PATH`:
 
 ## Choose ruby version
 
+If you specify a ruby version in your `gems.rb` of `Gemfile`, we'll use it, otherwise; keep reading.
+
 On your Clever Cloud application create an environment variable `RUBY_VERSION=rubyversion` where `rubyversion` represents:
 
  * "2" will select the greatest "2.X.Y" version available.
- * "2.3" will select the greatest "2.0.Y" version available.
+ * "2.3" will select the greatest "2.3.Y" version available.
```

Comment:
Later on, you say that the default version you provide is `2.4.Y`. 
What about using `2.4` as example here (and the line below)?